extensions/binding/org.eclipse.smarthome.binding.lifx/.classpath

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/lifx-sdk-java-1.0.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry excluding="org.eclipse.smarthome.binding.lifx.tim*" kind="src" path="src/main/java"/>
```

Comment:
I have no clue... Do not remember fiddling with that. The other files to be removed are because of a completely f*cked up commit from my side. It was still on the to-do list.